internal/distro/rhel8/distro.go

```diff
@@ -906,6 +939,9 @@ func New() distro.Distro {
 		distro: &r,
 		name:   "s390x",
```

Comment:
s390 seems to be missing `dracut-config-generic` which will lead to a host-only initramfs which does not contain the drivers for virtual disks, making it impossible to boot in qemu:

```
bootloaderPackages: []string{
   "dracut-config-generic",
   "s390utils-base"
}
```
I think  "s390utils-base", which contains `zipl`, is already being pull in, but maybe better be explicit about it.

Reply:
This was now added.